src/components/backend-ai-session-launcher.ts

```diff
@@ -2746,6 +2794,98 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
     this.shadowRoot.querySelector('#HPCOptimizationOptions').style.display = isOpenMPChecked ? 'none' : 'block';
   }
 
+  /**
+   * Toggle startup code input section according to session type
+   *
+   * @param {Event} e
+   */
+  _toggleStartUpCommandEditor(e) {
+    this.sessionType = e.target.value;
+    const startUpCommandEditor = this.shadowRoot.querySelector('#batch-mode-config-section');
+    startUpCommandEditor.style.display = (this.sessionType === 'batch') ? 'inline-flex' : 'none';
```

Comment:
When `startUpCommandEditor` starts to appear, `command-editor` just looks like a black block.
I think it's better to show up with an index.
Here's the way to show an index.
```javascript
const editor = this.shadowRoot.querySelector('#command-editor');
editor.refresh();
```
[Result Screenshot]
![image](https://user-images.githubusercontent.com/28584164/148484960-0ff8b079-ef03-4ebf-a58e-09cf1896f845.png)

Reply:
@Sujin-Kim1 I agree. And I also added autofocus when the session type is batch mode.
Could you check it again?

Reply:
It works properly👍🏻